src/components/ProjectList/ExampleProjects/ExampleProject.js

```diff
@@ -8,7 +8,7 @@ function ExampleProject({ project: { name, skills = [] } = {}, open = true }) {
   return (
     <Card className="primaryCard" key={name}>
       <Card.Header className="h4" aria-controls="task-item-contents" aria-expanded={open} onClick={() => setCardOpen(!cardOpen)}>
-        <h1>{name}</h1>
+        <h1 onClick={() => setCardOpen(!cardOpen)}>{name}</h1>
         <div className={`arrow ${cardOpen ? "open" : ""}`}></div>
```

Comment:
The onClick is in the Card.Header and h1, I presume that you found that was needed? Also does it open if you click the arrow itself?

Reply:
Thought it'd be nice for the user to be able to open the card when project title is also clicked and yeah card opens up when the arrow is clicked as well